src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -44,25 +44,48 @@
 from src.proto.grpc.testing import messages_pb2
 from src.proto.grpc.testing import test_pb2
 
+_INITIAL_METADATA_KEY = "x-grpc-test-echo-initial"
+_TRAILING_METADATA_KEY = "x-grpc-test-echo-trailing-bin"
+
+# If the request came with metadata in the context, this
+# helper function will copy it to the returned metadata.
+def _maybe_echo_metadata(context):
+  metadata = dict(context.invocation_metadata())
+  if _INITIAL_METADATA_KEY in metadata:
+    initial_metadata = (
+        _INITIAL_METADATA_KEY, metadata[_INITIAL_METADATA_KEY])
+    # Send initial metadata expects a list of tuples
+    context.send_initial_metadata([initial_metadata])
+  if _TRAILING_METADATA_KEY in metadata:
+    trailing_metadata = (
+      _TRAILING_METADATA_KEY, metadata[_TRAILING_METADATA_KEY])
+    # Set trailing metadata expects a list of tuples
+    context.set_trailing_metadata([trailing_metadata])
+
+# If the request asks for a particular status code and details, this helper
+# sets those values. It returns true if a status code was requested, because
+# the server does not continue processing in that case
+def _maybe_echo_status_and_message(request, context):
```

Comment:
If the type of this second parameter is `grpc.ServicerContext`, go ahead and give the parameter a name that reminds the reader of that.